src/tablet/drop_procedure_before_drop_table_test.cc

```diff
@@ -125,160 +125,35 @@ void ShowTable(::openmldb::RpcClient<::openmldb::nameserver::NameServer_Stub>& n
     ::openmldb::nameserver::ShowTableRequest request;
     ::openmldb::nameserver::ShowTableResponse response;
     request.set_db(db);
-    request.set_show_all(true);
+    request.set_show_all(false);
     bool ok = name_server_client.SendRequest(&::openmldb::nameserver::NameServer_Stub::ShowTable, &request, &response,
                                              FLAGS_request_timeout_ms, 1);
     ASSERT_TRUE(ok);
     ASSERT_EQ(response.table_info_size(), size);
 }
 
-TEST_F(SqlClusterTest, RecoverProcedure) {
+TEST_F(SqlClusterTest, DropProcedureBeforeDropTable) {
```

Comment:
Why rename this test file? `drop_procedure_before_drop_table_test.cc` is not a good name

Reply:
I need to separate the RecoverProcedure and DropProcedureBeforeDropTable for the tests to pass. There's probably some bugs in the tests that we can fix later